pkg/arvo/sys/vane/ames.hoon

```diff
@@ -728,6 +782,7 @@
 ::
 =<  =*  adult-gate  .
     =|  queued-events=(qeu queued-event)
+    =|  cached-state=[?(%5 %~) ames-state-5]
```

Comment:
Is there a reason this isn't 
```
=|  cached-state=(unit [%5 ames-state-5])
```
?

Reply:
None, I like yours more—I remember the type system yelling at me when I tried that, but just now I just added `?>  ?=(^ cached-state)` prior to the `state-5-to-6` update and it doesn't complain any more :)